v7/publication_list/README.md

```diff
@@ -0,0 +1,57 @@
+# Publication List
+
+A Nikola plugin to easily manage publication list.
+
+This plugin provides a directive `publication-list`, which reads a bib file and
+display the references in them on the web page. The `publication-list` directive
+accepts one option `:style:`, which is the style of the bibliography. All
+available styles are provided by Pybtex. You can see the [list of styles]() in
```

Comment:
Broken link.

    You can see the [list of styles][] in the Pybtex repository.